rollup.config.js

```diff
@@ -9,12 +9,34 @@ export default [
       ...Object.keys(pkg.peerDependencies),
     ],
     output: [
-      { file: pkg.main, format: 'cjs' },
-      { file: pkg.module, format: 'es' },
+      {
+        file: pkg.main,
+        format: 'cjs',
+        plugins: [getBabelOutputPlugin({
+          presets: [['airbnb', {
+            modules: false,
+            runtimeVersion: '7.12.5',
+            runtimeHelpersUseESModules: false,
+          }]],
+        })],
+      },
+      {
+        file: pkg.module,
+        format: 'es',
+        plugins: [getBabelOutputPlugin({
+          presets: [['airbnb', {
+            modules: false,
+            runtimeVersion: '7.12.5',
+            runtimeHelpersUseESModules: true,
+          }]],
+        })],
+      },
     ],
     plugins: [
       babel({
+        babelrc: false,
```

Comment:
I think this means that we probably want to remove the production section of the .babelrc. I can do that separately.